src/N98/Magento/Command/Developer/Module/Rewrite/ConflictsCommand.php

```diff
@@ -167,6 +171,48 @@ protected function _isInheritanceConflict($classes)
     }
 
     /**
+     * Check if one of the rewritten classes actually is written to fix the conflict.
+     * We can check by checking for an annotation in the PHPDOC block (@SolvesRewriteConflict)
+     *
+     * @var array $rewriteClasses
+     * @var string $rewrittenClass
+     * @return bool
+     */
+    protected function _isARewriteFixingTheConflict($rewriteClasses, $rewrittenClass)
+    {
+        foreach ($rewriteClasses as $rewriteClass) {
+            $r = new \ReflectionClass($rewriteClass);
+            $doc = $r->getDocComment();
+            preg_match_all('#@(.*?)\n#s', $doc, $annotations);
+
+            foreach ($annotations as $annotation) {
+                if (stripos($annotation[0], 'SolvesRewriteConflict') === 0) {
+                    preg_match('/SolvesRewriteConflict\((.*)\)/', $annotation[0], $matches);
+
+                    $classesInAnnotation = explode(',', $matches[1]);
+                    $firstClassInAnnotation = array_shift($classesInAnnotation);
+                    $rewriteClassesWithoutCurrentClass = array_diff($rewriteClasses, array($rewriteClass));
+
+                    /* Check if all the conflicted classes are actually mentioned in the annotation,
+                     * another conflict could later have been introduced.
+                     */
+                    if (
+                        count($classesInAnnotation) > 1
+                        &&
```

Comment:
You can merge this line with the next one, so it's one line with the operator in front. Same then for the next-next and next-next-next line :)